docs/USER_RECORD.md

```diff
@@ -248,6 +252,13 @@ sessions.
 purpose of an avatar. This must be a string, and should follow the semantics
 defined in the [Icon Naming
 Specification](https://standards.freedesktop.org/icon-naming-spec/icon-naming-spec-latest.html).
+You should probably use `avatarPath` instead.
+
+`avatarPath` → The path to a file containing the user's avatar image, a string.
+If the path is relative, it will be resolved relative to the user's bulk
+directory. The file referenced by this setting should be world-readable
+(i.e. should have a mode of 0644). If both `avatarPath` and `iconName` are
+set, `avatarPath` should take priority.
```

Comment:
hmm, do we really want to do this like this? given that images are supposed to be portable i think we should not allow absolute paths here at all, because they only make sense relative to some system.

And we actually want that any user management system copies the avatar picture into the home dir, so it is truly portable, so we really want to take away the ability to reference external reosurces.

But I wonder, should this really be a json property at all? i.e. do we need the indirection? because we could just say the avatar is placed in a file "avatar-image" or so in the user's bulk dir. (without any suffix, so that PNG, JPEG, whatever is actually fine, parsers would have to deal with that).

i mean, it appears to me that this bulk dir mostly contains "blobs", i.e. data that is relatively unstructured from homed's pov, hence it should bs sufficient to give the files generic identifiers via filenames and that's all.

or in other words, instead of adding a new json property here to the spec, add a new section to the bottom, which lists "well-known" file names in the bulk dir, the avatar being the most prominent of them.

Reply:
I like this concept. Will rework